rafthttp/transport.go

```diff
@@ -37,6 +38,13 @@ type Raft interface {
 	ReportSnapshot(id uint64, status raft.SnapshotStatus)
 }
 
+type SnapshotStore interface {
```

Comment:
I think you told me that you will put this into etcdserver/snapshotStore or something like that.